redmine/redmine_dumper.py

```diff
@@ -0,0 +1,33 @@
+import sys
+from redminelib import Redmine
+import redmine.config as config
+from db.models import Base, Issue
+import logging
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+
+class RedmineDumper:
+
+    def __init__(self) -> None:
+        self.logger = logging.getLogger(self.__module__)
+        self.logger.setLevel(logging.DEBUG)
+        handler = logging.StreamHandler(sys.stdout)
+        handler.setLevel(logging.DEBUG)
+        formatter = logging.Formatter(
+            '%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+        handler.setFormatter(formatter)
+        self.logger.addHandler(handler)
```

Comment:
Maybe all this logging initialization could be moved to a library because will be used in other scripts

Reply:
agree but I prefer to do it as refactoring in the future when we will have certain class which will require this logging than it will be more clear how exactly to do this extraction